assets/airports/kstl.json

```diff
@@ -0,0 +1,6728 @@
+                "30R": [["PYUNG", "A25+"], "ALLNT", "ELDBB"]
+            },
+            "body": ["CHUUC"],
+            "exitPoints": {
+                "ARG": ["MYERZ", "ARG"],
+                "MAW": ["MYERZ", "MAW"],
+                "MYERZ": ["MYERZ"]
+            },
+            "draw": [
+                ["GEJAY", "CHUUC", "MYERZ*"],
+                ["WAARN", "ELDBB", "CHUUC"],
+                ["MYERZ", "ARG*"],
+                ["MYERZ", "MAW*"]
+            ]
+        },
+        "DRUSE3": {
+            "icao": "DRUSE3",
+            "name": "Droose Three",
+            "rwy": {
+                "6": [],
+                "11": [["LUUBE", "A25+"]],
+                "12L": [["HEFNR", "A25+"]],
+                "12R": [["HEFNR", "A25+"]],
+                "24": [],
+                "29": [["HAGEE", "A25+"], "WAARN", "ELDBB", "BERAA"],
+                "30L": [["PYUNG", "A25+"], "ALLNT", "ELDBB", "BERAA"],
+                "30R": [["PYUNG", "A25+"], "ALLNT", "ELDBB", "BERAA"]
+            },
+            "body": ["DRUSE"],
+            "exitPoints": {
+                "BNA": ["PLESS", "DENNI", "TINGS", "LAJUG", "BNA"],
+                "DENNI": ["PLESS", "DENNI"]
+            },
+            "draw": [
+                ["HEFNR", "DRUSE", "PLESS", "DENNI*"],
+                ["DENNI", "TINGS", "LAJUG", "BNA*"],
+                ["ELDBB", "BERAA", "DRUSE"],
+                ["LUUBE", "DRUSE"]
+            ]
+        },
+        "EEMOS3": {
+            "icao": "EEMOS3",
+            "name": "Emose Three",
+            "rwy": {
+                "6": [],
+                "11": [["FUNKE", "A25+"], "BEART", "GMSKI"],
+                "12L": [["FUNKE", "A25+"], "BEART", "GMSKI"],
+                "12R": [["FUNKE", "A25+"], "BEART", "GMSKI"],
+                "24": [],
+                "29": [["DSAPP", "A25+"], "CONRR"],
+                "30L": [["WOLTR", "A25+"], "CONRR"],
+                "30R": [["WOLTR", "A25+"], "CONRR"]
+            },
+            "body": ["EEMOS"],
+            "exitPoints": {
+                "NEENS": ["NEENS"]
+            },
+            "draw": [
+                ["WOLTR", "CONRR", "EEMOS", "NEENS*"],
+                ["FUNKE", "BEART", "GMSKI", "EEMOS"],
+                ["DSAPP", "CONRR"]
+            ]
+        },
+        "GATWY8": {
+            "icao": "GATWY8",
+            "name": "Gateway Eight",
+            "rwy": {
+                "6": [],
+                "11": [],
+                "12L": [],
+                "12R": [],
+                "24": [],
+                "29": [],
+                "30L": [],
+                "30R": []
+            },
+            "body": ["CSX", "TWILA"],
+            "exitPoints": {
+                "BIB": ["JIGSY", "BIB"],
+                "VHP": ["JIGSY", "BIB", "WORKE", "KELLY", "VHP"],
+                "CREEP": ["JIGSY", "BIB", "WORKE", "KELLY", "SHB", "CREEP"],
+                "JIGSY": ["JIGSY"],
+                "ROD": ["JIGSY", "BIB", "WORKE", "KELLY", "SHB", "ROD"]
```

Comment:
Order alphabetically (I know, the charts list them by spoken name, hence the order here :wink:)